### PR TITLE
feat: malfunction mania blog post

### PR DESCRIPTION
**Pull request by ElTimuro**

### CODE REVIEW

File: apps/marketing/content/blog/manfunction-mania.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+As Documenso 1.0 just hit our test environment, and we are calling a “MALFUNCTION MANIA”. A big, public testing phase, where we invite everyone to try out the new version, hunt down, report and fix any and all malfunctions (aka bugs) and give feedback before release. Malfunction Mania will happen alongside our own internal testing and by combining the two, we want to ensure the best possible release we can have for Version 1.0. We know that many of you have been eager to contribute and this is your chance (the first of many to come).
+
+## As part of Malfunction Mania, we are offering special bounties:
+
+- $50 for reporting serious issues in a reproduceable way
+- $100 for fixing reported or other critical issues via accepted PR
+- Exclusive Merch: Every worthy contribution to Malfunction Mania will be awarded the Limited Edition “Signature Look” Shirt
```
Comment (Mythie) [inline apps/marketing/content/blog/manfunction-mania.mdx:39]: I think we should have some more nuance on this since not all bugs are the same.

Comment (ElTimuro) [reply apps/marketing/content/blog/manfunction-mania.mdx:39]: @catalinpit @dguyen @dguyen thoughts?

**Review by fmerian — COMMENTED** [review]
made minor edits for consistency. LGTM!

**Comment by catalinpit** [comment]
I'd advise running the article through Grammarly (or a similar tool). Other than that, it looks good!

**Comment by fmerian** [comment]
> I'd advise running the article through Grammarly (or a similar tool).

done!

File: apps/marketing/content/blog/manfunction-mania.mdx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+## As part of Malfunction Mania, we're offering special bug bounties
+
+- We award $25 - $100 per report/ issue/ fix, depending on the severity and if the problem is already known
+- Bounties will be awarded for fixing reported or other critical issues via accepted Pull Requests (PR)
+- Just reporting issues in a reproducible way can also be awarded
+- Smaller but notable contributions like minor issues and documentation will be awarded with exclusive merch as we see fit.
+
+## What you can do
+
+- Head over to the [testing environment](https://documen.so/test), check out the new version and give it a spin;
```
Comment (dephraiim) [inline apps/marketing/content/blog/manfunction-mania.mdx:44]: `;` at the end

File: apps/marketing/content/blog/manfunction-mania.mdx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+- Check out the [source code](https://github.com/documenso/documenso) on GitHub and look it over;
```
Comment (dephraiim) [inline apps/marketing/content/blog/manfunction-mania.mdx:45]: Same for this, `;` at the end

**Review by dephraiim — CHANGES_REQUESTED** [review]

**Pull request merged**